### ISSUE TITLE
Update profiling to authenticate using user password

### ISSUE THREAD

**Issue by janeeisenstein** [simplworld/simpl-modelservice#39]
The current profiling creates user sessions based solely on email address. The v1.0-RC authentication also requires supplying a password.

This ticket covers updating the profiling system to authenticate based on both email and password.

**Comment by janeeisenstein** [comment]
@frankwiles Am I right to assume a password will need to be passed to work with your authentication upgrade? I'm in the midst of writing a blog post on profiling and I'd prefer it not be out of date when the authentication upgrade is released. Would it make sense for me to I add an optional 'password' argument to the 'publish_as' and 'call_as' methods of ProfileCase on the master branch? Then the blog post could document using the password argument.

**Comment by frankwiles** [comment]
Yes password is going to be required for auth for all connections going forward so I think adding it as optional until 1.0 lands makes sense but then it shouldn't be optional.

**Comment by janeeisenstein** [comment]
Added user_password as a required parameter in version v0.10.0.
